package.json

```diff
@@ -50,7 +50,8 @@
   "devDependencies": {
     "@silverstripe/eslint-config": "^1.3.0",
     "@silverstripe/webpack-config": "^3.0.0-alpha2",
-    "@testing-library/react": "^14.0.0",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/react": "^16.1.0",
```

Comment:
Why is a new dependency being added?

Reply:
From v16 `@testing-library/dom was moved to a peer dependency and needs to be explicitly installed.` https://github.com/testing-library/react-testing-library/releases/tag/v16.0.0